src/MSBuild/OutOfProcTaskHostNode.cs

```diff
@@ -1014,6 +1016,39 @@ private void RunTask(object state)
                     _taskCompleteEvent.Set();
                 }
             }
+
+            /// <summary>
+            /// Inline tasks may have dependencies that were resolved during TaskFactory initialization.
+            /// Recreate assembly resolve handlers for these dependencies.
+            /// </summary>
+            static void RegisterDependencyLoadHandlersFromManifest(string taskLocation)
```

Comment:
Does this have to be distinct from the one in `TaskFactoryUtilities`? Feels a bit repetitive.